index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
+  .header_container {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+
+    /* left: 0px;
+    top: 0px; */
+
+    background: #3c3a39;
+    color: red;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .item {
+    align-self: center;
+  }
+
+  .name {
+    font-family: Roboto, serif;
```

Comment:
Please first add the font used in Figma template to your CSS file. You can follow this link for [Roboto](https://fonts.google.com/specimen/Roboto#styles).